src/pages/volunteers/[volId].tsx

```diff
@@ -69,4 +123,114 @@ export async function getStaticPaths() {
     return { paths, fallback: true };
 }
 
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        container: {
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+            margin: "45px",
+        },
+        volInfoContainer: {
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "center",
+            marginTop: "40px",
+            justifyContent: "space-between",
+            width: "100%",
+            [theme.breakpoints.between(0, "sm")]: {
+                flexDirection: "column",
+                width: "100%",
+            },
+        },
+        volInfoRight: {
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "space-between",
+            [theme.breakpoints.between(0, "sm")]: {
+                marginTop: "20px",
+                width: "90%",
+            },
+        },
+        nameCard: {
+            border: `2px solid ${theme.palette.primary.light}`,
```

Comment:
How do these borders look as 1px? They seem a little bit thick for some reason. Maybe try 1px and our primary color? If that doesn't look good then you can keep this - I'll leave it up to you!

Reply:
Good call! Updated the borders to 1px but left it the light color. I tried the primary main, but I think it looked a little off. Here are the new borders!

<img width="1280" alt="Screen Shot 2021-03-19 at 10 00 50 AM" src="https://user-images.githubusercontent.com/58822572/111792153-5a85ee80-889a-11eb-8067-a04b5635cb2f.png">